generators/client/templates/angular/src/main/webapp/app/admin/metrics/metrics.service.ts.ejs

```diff
@@ -22,16 +22,21 @@ import { Observable } from 'rxjs';
 
 import { SERVER_API_URL } from 'app/app.constants';
 
+export type MetricsKey = 'jvm' | 'http.server.requests' | 'cache' | 'services' | 'databases' | 'garbageCollector' | 'processMetrics';
+export type Metrics = { [key in MetricsKey]: any };
+export type Thread = any;
```

Comment:
That's impossible to type ?

Reply:
It's possible, but as this is used inside `ng-jhipster` then I decided not to do that to keep code light, not to pollute code with unused information.